.github/CODEOWNERS

```diff
@@ -318,6 +318,8 @@
 
 /modules/services/mpdris2.nix                         @pjones
 
+/modules/services/mpd-discord-rpc.nix		      @Kranzes
```

Comment:
I think this isn't indented enough. Either used tabs instead of spaces or vice-versa.

Reply:
Fixed now.